pkg/helm/controller/reconcile.go

```diff
@@ -261,7 +262,7 @@ func (r HelmOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.
 			}
 		}
 
-		log.Info("Updated release")
+		log.Info(fmt.Sprintf("Updated release, force: %t", force))
```

Comment:
Nit: Preferred format for structured logs would be:
```suggestion
		log.Info("Updated release", "force", force)
```